### ISSUE TITLE
Support URLs at bibleserver.com

### ISSUE THREAD

**Issue by sebastiankuhnert** [chadparry/bibleref#9]
bibleserver.com offers a service similar to biblegateway.com, providing a different set of Bible translations and UI languages.

**Comment by chadparry** [comment]
Agreed. I didn't want to be biased towards any one website. Having support for two servers will be fairer.

It looks like the bibleserver.com API only supports references within a single chapter. For example, it can't support "Mal. 3-4" as a single reference. That would have to be broken up into two links, to Malachi 3 and 4 separately. I had to do the same thing in bibleref-lds for the lds.org website. It wasn't easy, but we can employ a similar trick in bibleref.
